DESCRIPTION

```diff
@@ -12,12 +12,14 @@ Imports:
     digest (>= 0.6.8),
     DT, 
     shiny (>= 0.13.2.9003),
-    shinyjs (>= 0.5.2)
+    shinyjs (>= 0.5.2),
+    checkmate
```

Comment:
I prefer not to introduce this package, unless it has some uses that really can't be easily done without it